pkg/controllers/provisioning/provisioner.go

```diff
@@ -141,8 +143,17 @@ func (p *Provisioner) getPods(ctx context.Context) ([]*v1.Pod, error) {
 	}
 	var pods []*v1.Pod
 	for i := range podList.Items {
-		if isProvisionable(&podList.Items[i]) {
-			pods = append(pods, &podList.Items[i])
+		pod := podList.Items[i]
+		errs := multierr.Combine(
+			validate(&pod),
+			p.volumeTopology.validatePersistentVolumeClaims(ctx, &pod),
+		)
+		if errs != nil {
+			logging.FromContext(ctx).With("pod", fmt.Sprintf("%s/%s", pod.Namespace, pod.Name)).Errorf("Unable to batch pod, %s", errs)
```

Comment:
I don't think we want the errors logged here.  This error will just be repeated for a bad pod a ton of times if you have lots of pod churn.

Reply:
You're right it will log at every attempt to batch pods.  But we can't necessarily return the error otherwise we will fail the entire batch of pods.  Or are you suggesting we just make it a debug log instead?

Reply:
Does the error show up on the pod events from kube-scheduler? If so, maybe we don't need to log anything and just consider it a non-provisionable pod.

Reply:
yep, it does.  I was thinking either Debug level logging, or don't log at all.
```
Events:
  Type     Reason            Age              From               Message
  ----     ------            ----             ----               -------
  Warning  FailedScheduling  7s (x2 over 8s)  default-scheduler  0/1 nodes are available: 1 persistentvolumeclaim "myclaim" not found.
```

Reply:
Either sounds good to me too.